cmd/vic-machine/create/create.go

```diff
@@ -485,7 +485,7 @@ func (c *Create) Flags() []cli.Flag {
 
 		cli.BoolFlag{
 			Name:        "force, f",
-			Usage:       "Force the install, removing existing if present",
```

Comment:
I would also keep the clarification that it will install over an existing deployment in the error message 😄